.github/ISSUE_TEMPLATE.md

```diff
@@ -14,7 +14,7 @@
 
 ## Steps to Reproduce (for bugs)
 <!--- Provide a link to a live example, or - -->
-<!--- Fork the jsfiddle to reproduce https://jsfiddle.net/uq8osbp1/ ->
+<!--- Fork the jsfiddle to reproduce https://codepen.io/alphalpha/pen/oqKJgG ->
```

Comment:
Nit, could you change the text to say codepen?  we must have missed that before.

Reply:
Oops. 
Will do.